pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_crd_reconcile.go

```diff
@@ -51,10 +52,8 @@ var (
 
 	// removed CRD StoredVersions
 	removedCRDStoredVersions = map[string]string{
-		"placements.cluster.open-cluster-management.io":                "v1alpha1",
-		"placementdecisions.cluster.open-cluster-management.io":        "v1alpha1",
-		"managedclustersets.cluster.open-cluster-management.io":        "v1alpha1",
-		"managedclustersetbindings.cluster.open-cluster-management.io": "v1alpha1",
+		"managedclustersets.cluster.open-cluster-management.io":        "v1beta1",
```

Comment:
do we still need creating migration resource for placement/clusterset/clustersetbinding?

Reply:
nvm, I see it